### PR TITLE
Fix/updated task media upload youtube

### PR DESCRIPTION
**Pull request by diferpaho**

### CODE REVIEW

File: djangoplicity/media/tasks.py [hunk 1]
```diff
@@ -600,7 +600,7 @@ def upload_youtube(video_id, user_id=None):
 
     def mail_user(subject, body=''):
         if user and user.email:
-            send_mail(subject, body, 'no-reply@eso.org', [user.email])
+            send_mail(subject, body, 'no-reply@esahubble.org', [user.email])
```
Comment (EdisonArango) [inline djangoplicity/media/tasks.py:603]: Nooo!!!!, don't update this to specific projects!, this is being used by multiple organizations, just update this to send mail to managers or send to admins!!

Comment (danialeresbar) [reply djangoplicity/media/tasks.py:603]: That's right :( 
@diferpaho  This project is the core of other projects. Any update here will be reflected in those projects too.

Comment (EdisonArango) [inline djangoplicity/media/tasks.py:603]:
```suggestion
            send_mail(subject, body, settings.DEFAULT_FROM_EMAIL, [user.email])
```

See: https://docs.djangoproject.com/en/2.2/topics/email/

**Comment by danialeresbar** [comment]
This pull request is not longer need

**Pull request closed without merging**